android-core/src/androidTest/kotlin/com.mparticle/MParticleTest.kt

```diff
@@ -302,6 +305,46 @@ class MParticleTest : BaseCleanStartedEachTest() {
         }
     }
 
+    @Test
+    fun testWrapperSdkVersionWithoutBeingSet() {
+        val instance = MParticle.getInstance()!!
```

Comment:
is there a better way to do this instead of having so many `!!` in your tests?  or maybe we clean it up afterwards in a future PR?

Reply:
The instance is nullable by definition. For sure in the code is a horrible practice to force unwrap nullables, in this case the test class on the @Before is making sure that the instance is created, so in this particular situation using the !! is checking only relying on a pre-requisite.
If you want I can change it, using a  ?.let block - but going within the nullable situation should never happen and there it did, I would prefer the test to fail throwing an exception.